plugins/modules/ipagroup.py

```diff
@@ -243,28 +253,35 @@ def main():
     if state == "present":
         if len(names) != 1:
             ansible_module.fail_json(
-                msg="Only one group can be added at a time.")
+                msg="Only one group can be added at a time."
+            )
         if action == "member":
-            invalid = ["description", "gid", "nonposix", "external",
-                       "nomembers"]
+            invalid = [
+                "description",
+                "gid",
+                "nonposix",
+                "external",
+                "nomembers",
+            ]
```

Comment:
This is one type of change that I do not like. It make elements disperse, and they don't seem to be part of the same thing.